nixos/doc/manual/installation/installing.xml

```diff
@@ -7,11 +7,11 @@
  <section xml:id="sec-installation-booting">
   <title>Booting the system</title>
 
-  <para>
-   NixOS can be installed on BIOS or UEFI systems. The procedure for a UEFI
-   installation is by and large the same as a BIOS installation. The
-   differences are mentioned in the steps that follow.
-  </para>
+  <para>NixOS can be installed on systems using UEFI or MBR bios. The procedure for a UEFI
+   installation is by and large the same as a MBR installation. The differences are mentioned in the
+   steps that follow. If you are installing in a virtual machine check which type of BIOS it is
+   being used to be able to decide whether you can do an UEFI or a MBR installation. VMWare for
```

Comment:
"check which type of BIOS it is being used" → "check which type of firmware is used"

Phrasing of *it is being used* changed to *is used*, and using the term *firmware* as this is what both VMware and VirtualBox call the option. Using *BIOS* here is bound to be confusing with Legacy BIOS boot mode.